platform/broadcom/sonic-platform-modules-dell/common/dell_pmc.c

```diff
@@ -1738,24 +1743,39 @@ static ssize_t show_psu(struct device *dev,
 
 /* FAN and PSU EEPROM PPID format is:
    COUNTRY_CODE-PART_NO-MFG_ID-MFG_DATE_CODE-SERIAL_NO-LABEL_REV */
-static ssize_t show_psu_ppid(struct device *dev,
+static ssize_t show_ppid(struct device *dev,
                 struct device_attribute *devattr, char *buf)
 {
 	int index = to_sensor_dev_attr(devattr)->index;
 	struct smf_data *data = dev_get_drvdata(dev);
-	char psu_ppid[EEPROM_PPID_SIZE + 1] = {0};
+	char ppid[EEPROM_PPID_SIZE + 1] = {0};
 	char psu_mfg_date[EEPROM_MFG_DATE_SIZE + 1] = {0};
 	char psu_mfg_date_code[EEPROM_DATE_CODE_SIZE + 1] = {0};
 	char temp;
 	int i, reg, ret = 0, ppid_pos = 0;
 
 	switch(index) {
 
-		case 1:
-			// PPID starts from Country Code
+        /* PPID starts from Country Code*/
+        case 0:
+            reg = FAN_1_SERIAL_CODE;
+            break;
+        case 1:
+            reg = FAN_2_SERIAL_CODE;
+            break;
+        case 2:
+            reg = FAN_3_SERIAL_CODE;
+            break;
+        case 3:
+            reg = FAN_4_SERIAL_CODE;
+            break;
+        case 4:
+            reg = FAN_5_SERIAL_CODE;
+            break;
+		case 10:
```

Comment:
Cases 10 and 11 indented too far, not aligned with others

Reply:
Thanks for your valuable feedback ...
Corrected the alignment ...